### PR TITLE
Updated versioner truth data

### PR DESCRIPTION
**Pull request by AgoinsUSGS**
Updated the ControlNetVersioner truth data.

### CODE REVIEW

**Comment by AgoinsUSGS** [comment]
[tests failing - control net versioner](https://trello.com/c/6miDMVKf/158-tests-failing-control-net-versioner)

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.truth [hunk 1]
```diff
@@ -1,11 +1,12 @@
 Test ControlNetVersioner
-Reading: ./reallyOldNetwork.net...
+Reading: $control/testData/unitTest_ControlNetVersioner_PvlNetwork2_PvlV0001.net...
 
 Read network...
-**I/O ERROR** Reading the control network [reallyOldNetwork.net] failed.
-**I/O ERROR** Unable to get convert ControlNet Version 1 to Version 2.
+**I/O ERROR** Reading the control network [unitTest_ControlNetVersioner_PvlNetwork2_PvlV0001.net] failed.
+**I/O ERROR** Failed to initialize control point at index [0].
```
Comment (kberryUSGS) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.truth]: Just want to double-check that this line should definitely still be in here?

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.truth [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+**I/O ERROR** Unable to get target body radii for [] when calculating covariance matrix.
```
Comment (kberryUSGS) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.truth]: Just want to double-check that this line should definitely still be in here?

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.truth (hunk 1 above)
Comment (AgoinsUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.truth]: These are thrown from ControlNetVersioner::CreatePoint. The internal errors that were called are passed up the chain and ultimately displayed with the "Reading control network [x] failed" message, but we can throw that message without displaying the errors that caused it to fail if they're unneccessary output. Should we only display the latest error message?

Comment (kberryUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.truth]: I think this is okay as is, I just wanted to double-check, since it changed.

**Pull request merged**